libs/langchain/langchain/llms/bedrock.py

```diff
@@ -33,6 +34,21 @@ def prepare_input(
 
         return input_body
 
+    @classmethod
+    def prepare_output_stream(
+        cls, provider: str, response: Any
+    ) -> Iterator[GenerationChunk]:
+        stream = response.get("body")
+        if stream:
+            for event in stream:
+                chunk = event.get("chunk")
+                if chunk:
+                    chunk_obj = json.loads(chunk.get("bytes").decode())
+                    if provider == "anthropic":
+                        yield GenerationChunk(text=chunk_obj["completion"])
+                    else:
+                        yield GenerationChunk(text=chunk_obj["outputText"])
```

Comment:
If a new provider is introduced with a different output key we don't expect here, is it better to throw a more meaningful error instead of leaving this as a future cryptic KeyError?

Reply:
This is for `amazon.titan` streaming output if I am not wrong. Also last I checked AI21's Jurassic models don't support streaming last I checked. Ideally this should throw an error.